src/components/personkort/PersonkortHeader/PersonkortHeaderTags.tsx

```diff
@@ -36,6 +42,12 @@ export const PersonkortHeaderTags = () => {
     tilrettelagtKommunikasjon?.talesprakTolk?.value;
   const tegnsprakTolkSprakkode =
     tilrettelagtKommunikasjon?.tegnsprakTolk?.value;
+  const isOnFriskmeldingVetak =
+    vedtakFriskTilArbeid[0] &&
+    isTodayWithinInterval(
+      vedtakFriskTilArbeid[0].fom,
+      vedtakFriskTilArbeid[0].tom
+    );
```

Comment:
Vet ikke om jeg skjønte predikatet her. Hvis det ikke er inkludert tror jeg det også er viktig å vite om det er et kommende 8-5 vedtak. Altså så lenge dagens dato er før eller det samme som `tom`, så bør vi ha label 😊
Da holder kanskje denne namingen også👇😊
```suggestion
  const isFriskmeldingVedtak =
    vedtakFriskTilArbeid[0] &&
    isTodayWithinInterval(
      vedtakFriskTilArbeid[0].fom,
      vedtakFriskTilArbeid[0].tom
    );
```

Reply:
Ja, good point! Det kan jo være man uansett vil vise det selv om selve perioden ikke har startet! Jeg skal høre med Stine. Skriveleif også, ser jeg 👀

Reply:
Yes, Stine er enig også, jeg endrer 🙌🏼

Reply:
Skal vi skal vise den så lenge det er et kommende eller aktivt vedtak? Og om det er ferdigbehandlet har ingen betydning? Antar ikke, bare kaster ut spørsmålet :)

Reply:
Tenkte å vise det fra vedtaket er innvilget og til perioden er over, uavh av ferdigbehandling ja. Tanken er at du skal raskt kunne se at det er noe på gang her som man burde ta høyde for i oppfølgingen.